docs/user_docs/cli/kbcli_login.md

```diff
@@ -0,0 +1,49 @@
+---
+title: kbcli login
+---
+
+Authenticate with the KubeBlocks Cloud
+
+```
+kbcli login [flags]
+```
+
+### Options
+
+```
+      --client-id string   The client ID for the KubeBlocks cloud application.
+  -h, --help               help for login
+      --no-browser         Do not open the browser for authentication.
+      --site string        The KubeBlocks Auth API Base URL. (default "https://tenent2.jp.auth0.com")
```

Comment:
Users should not be required to know which IDP provider Cloud uses. `kbcli` should interact with the Cloud login page, and the only thing users need to specify is the Cloud region.